### ISSUE TITLE
[Bug][a11y]: Youtube videos cannot be selected with Talkback

### ISSUE THREAD

**Issue by SoftVision-LorandJanos** [mozilla-mobile/fenix#27316]
### Steps to reproduce

1. Turn TalkBack (or the available screen reader) on.
2. Go to youtube.com and select any video (except the short ones).

### Expected behaviour

The video is selected.

### Actual behaviour

The video is not selected.

### Device name

Google Pixel 4

### Android version

Android 12

### Firefox release type

Firefox Nightly

### Firefox version

107.0a1

### Additional information

- The short videos are selected as expected.

https://user-images.githubusercontent.com/48829350/194401752-9a066e63-5000-4c25-97dc-8c796d0c3a42.mp4

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-21751)

**Comment by Mugurell** [comment]
This does not work in Chrome also:

https://user-images.githubusercontent.com/11428869/194498123-caa2e7b9-41f0-49a5-a20f-c45d8faf5620.mp4

**Comment by Mugurell** [comment]
Moving to Bugzilla for more investigations.

**Comment by Mugurell** [comment]
Moved to bugzilla: https://bugzilla.mozilla.org/show_bug.cgi?id=1794084

Change performed by the [Move to Bugzilla add-on](https://addons.mozilla.org/en-US/firefox/addon/move-to-bugzilla/).
